tonic/src/status.rs

```diff
@@ -555,6 +592,14 @@ fn invalid_header_value_byte<Error: fmt::Display>(err: Error) -> Status {
     )
 }
 
+impl TryFrom<Box<dyn Error + Send + Sync + 'static>> for Status {
+    type Error = Box<dyn Error + Send + Sync + 'static>;
+
+    fn try_from(err: Box<dyn Error + Send + Sync + 'static>) -> Result<Self, Self::Error> {
+        Status::try_from_error(err)
+    }
+}
+
```

Comment:
Is there a specific use-case this impl enables, or is mainly for convenience? I dunno, feels a bit out of place for me.

Reply:
I don't recall, and I can't find a relevant use of this in the main code bases in which I work. Shall I remove it?

Reply:
Yeah I think we should just remove it.